components/profile.js

```diff
@@ -1,6 +1,7 @@
 const Cheerio = require('cheerio');
 const FS = require('fs');
 const SteamID = require('steamid');
+const Param = require('jquery-param');
```

Comment:
Do we really need this?

Reply:
I tried the original way of doing that, but you can't have multiple `profile_showcase[]` as json keys, so I did it the same way as the `serializeArray()` output, but this didn't work as a parameter, so it needed to be submitted as a string to the function, and the code originally was only implemented for my own project, so the fastest way was using the param function of JQuery

Reply:
I really would like to avoid adding a dependency for this.